wordpress-mobile/WordPress-Android
# EventBus: updated the Stories library with latest EventBus v3.2.0 version change

Updates the Stories library to its latest commit hash, which contains the latest EventBus version [upgrade](https://github.com/Automattic/stories-android/pull/654)

Related PRs:
- WPAndroid PR https://github.com/wordpress-mobile/WordPress-Android/pull/14272
- Stories PR: https://github.com/Automattic/stories-android/pull/654

To test (smoke test)
1. create a new Story few slides with the WPAndroid app
2. tap on the finish button on the right-top corner of the screen
3. optionally enter some post title and Publish
3. verify the snackbars for Uploading / saved appear (these are connected through EventBus)

PR submission checklist:
